src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1

```diff
@@ -126,9 +127,52 @@ Licensed under the MIT License. See License.txt in the project root for license
             # IconUri = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = '## 2018.10.28
-            * Bug fixes
+
+            ReleaseNotes = '## 2018.11.5
+            * New cmdlet
+	            * Get-AzsDrive
+	            * Get-AzsVolume
+	            * Get-AzsStorageSubSystem
+            * Unsupported cmdlet
+	            * Get-AzsInfrastructureVolume
+            
+            ## 2018.10.28
+                * Bug fixes
                 * Fixed a bug in Add-AzsScaleUnitNode
+                
+            ## 2018.9.10
```

Comment:
Where did this come from?

Reply:
Now only retains the latest release notes.